### PR TITLE
add top nav, update bottom nav 

### PR DESCRIPTION
**Pull request by by-bai**
top navbar
- **displayBackButton** prop determines whether top nav comes with a back button or not
- /support and /profile 's top nav should not have a back button.
- note: currently before more screens are up, /support serves as a temporary demo for top nav with back button

bottom navbar
- imported icons from figma to /public/assets

### CODE REVIEW

**Comment by myyk** [comment]
This mostly looks really good and matching the designs. There are a couple issues I think:

 * (blocking merge) Something looks wrong with the page when it looses focus and then comes back. I think we should try to fix this before merging it bc it might make development on other parts difficult.
 * It feels a little weird to me that there's no back button for the Profile page like there is for the Support page.

If you get some time pls see if you can fix those issues especially the first one. Thanks~!

**Comment by by-bai** [comment]
Hey myyk, i’m not sure what you’re referring to for the first one, hope to get more details on that!

For the second point, as stated on my PR: note: currently before more screens are up, /support serves as a temporary demo for how a top nav with back button looks like. /profile and /support should not come with a back button since they can be accessed by bottom nav. only their subpages will have a back button. But if it makes things unclear, i’ll remove the back button for this merge!

Thanks :)

File: pages/support.tsx
Comment (myyk) [inline pages/support.tsx:9]: I think I would actually just comment here that this is `true` for demonstration purposes but should be false in a `//TODO: ` comment.

**Comment by myyk** [comment]
Oh yeah, about the back button you did mention that but I didn't remember by the time I launched a preview. I'd just add a code comment. 👍🏽 

>(blocking merge) Something looks wrong with the page when it looses focus and then comes back. I think we should try to fix this before merging it bc it might make development on other parts difficult.

I'm having trouble repro-ing it. Maybe it's ok and I had dirtied the commit before or something.

**Comment by myyk** [comment]
Oh sorry, I think with nothing blocking I can approve this.

**Review by myyk — APPROVED** [review]

**Comment by by-bai** [comment]
Thanks Myyk! will add clearer code comments in future.

**Pull request merged**
